tests/post_training/pipelines/masked_language_modeling.py

```diff
@@ -0,0 +1,88 @@
+# Copyright (c) 2023 Intel Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#      http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import numpy as np
+import onnx
+import torch
+import transformers
+from optimum.intel import OVQuantizer
+from optimum.intel.openvino import OVModelForSequenceClassification
+from optimum.onnxruntime import ORTModelForSequenceClassification
+
+import nncf
+from tests.post_training.pipelines.base import OV_BACKENDS
+from tests.post_training.pipelines.base import PT_BACKENDS
+from tests.post_training.pipelines.base import BackendType
+from tests.post_training.pipelines.base import BaseTestPipeline
+
+
+class MaskedLanguageModelingHF(BaseTestPipeline):
+    """Pipeline for masked language models from Hugging Face repository"""
+
+    def prepare_model(self) -> None:
+        if self.backend in PT_BACKENDS:
+            self.model_hf = transformers.AutoModelForSequenceClassification.from_pretrained(self.model_id)
+            self.model = self.model_hf
+            self.dummy_tensor = torch.Tensor([[0] * 53]).type(dtype=torch.LongTensor)
```

Comment:
Can we store the magic value 53 in a constant/variable?

Reply:
replaced to use dummy_inputs from HF model.